### DISCUSSION TITLE
Problem accessing release 20230521

### DISCUSSION THREAD

**Discussion by ceejayemm** [rern/rAudio#1302] [Bug Reports]
@rern 

I don't think the release process for this new release has happened properly. There are no release notes and my RPI does not 'see' a new release available although Github says there is. Is it still work in progress and I should wait awhile ?

Thanks for all your efforts with this project, this is NOT a dig but hopefully a helpful note.

Chris

**Comment by Raitsa** [comment]
Yeah same prob here, no way to online update.

**Reply by Lubos-V** [reply]
Exactly. The player can only be updated "normally" to the version of 10th March. For the development branch, there are updates of course, but I aso cannot see the latest release...

**Comment by klassikmann** [comment]
Can't confirm in rAudio 20230522

**Comment by ceejayemm** [comment]
Seems to be solved with 20230522

Thanks

Chris
